packages/polymath-ui/src/components/STOStatus/STOStatus.js

```diff
@@ -84,71 +85,72 @@ export default class STOStatus extends Component<Props> {
         <p className="pui-sto-status-contract">
           Contract {etherscanAddress(details.address)}
         </p>
-        <div
-          className={
-            'pui-sto-status-grow' +
-            (this.props.isStoPaused ? ' pui-paused' : '')
-          }
-        >
-          <div className="pui-sto-status-numbers">
-            <div>{fractionComplete}%</div>
-            <div className="pui-key-value">
-              <div>Cap</div>
-              {capText}
-            </div>
-          </div>
-          <Box mb="l">
-            <ProgressBar progress={fractionComplete / 100} />
-          </Box>
-          <div className="pui-sto-status-bottom-row">
-            <div className="pui-sto-status-dates">
-              <div className="pui-key-value">
-                <div>Start Date</div>
-                {dateFormat(details.start)}
-              </div>
+        <Grid gridTemplateColumns={['', '', '', '1fr minmax(250px, 1fr)']}>
```

Comment:
Just a question: This empty strings would go away in the future with breakpoints in a object right?